pylzy/tests/api/v1/test_startup.py

```diff
@@ -41,23 +42,29 @@ def test(a: str, *, b: File) -> str:
             get_logging_config(),
             serializers=ser.imports(),
             op=test,
-            args_paths=[(str, arg_file)],
-            kwargs_paths={"b": (File, kwarg_file)},
-            output_paths=[(str, ret_file)],
-            exception_path=(Exception, exc_file)
+            args_paths=[(str, arg_file_name)],
+            kwargs_paths={"b": (File, kwarg_file_name)},
+            output_paths=[(str, ret_file_name)],
+            exception_path=(Exception, exc_file_name)
         )
 
         startup.main(pickle(req))
 
-        with open(ret_file, "rb") as f:
+        with open(ret_file_name, "rb") as f:
             ret = ser.find_serializer_by_type(str).deserialize(f, str)
             self.assertEqual("42", ret)
 
+        os.close(arg_file)
```

Comment:
It is still not ok.
Who will clear it in case of an exception?
Either use try/finally or a context manager.